src/content/docs/network-performance-monitoring/setup-performance-monitoring/network-flow-monitoring.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Set up network flow data monitoring
+tags:
+  - Integrations
+  - Network performance monitoring
+  - Installation
+  - Setup
+metaDescription: Set up network flow data monitoring.
+---
+
+Set up your network devices so they send network data to New Relic One.
+
+## Prerequisites [#prerequisites]
+
+- A **New Relic account ID**. Read how to [find your account ID](/docs/accounts/accounts-billing/account-setup/account-id/).
+- An **Insights Insert Key**. Read how to [generate an Insights insert key](/docs/apis/intro-apis/new-relic-api-keys/#insights-insert-key).
+- [Docker](https://docs.docker.com/engine/install/) installed in your local machine.
+
+## Supported types of network flow data [#supported-network-flow-data-types]
+
+Kentik's integration supports four types of network flow data. When running the **ktranslate** image, you can specify which type you want to monitor using the `-nf.source` option.
+
+<Callout variant="important">
+The **ktranslate** image only supports monitoring one type of network flow data type at a time. If you want to monitor several types, each will require a container.
+</Callout>
+
+To check the equivalence among the network flow data type and the value you need to specify when running the image, see the following table:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "300px" }}>
+        Network flow data type
+      </th>
+      <th>
+        `-nf.source` value
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        IPFIX
+      </td>
+      <td>
+        `ipfix`
+      </td>
+    </tr>
+    <tr>
+      <td>
+        NetFlow version 5
+      </td>
+      <td>
+        `netflow5`
+      </td>
+    </tr>
+    <tr>
+      <td>
+        NetFlow version 9
+      </td>
+      <td>
+        `netflow9`
+      </td>
+    </tr>
+    <tr>
+      <td>
+        sFlow
+      </td>
+      <td>
+        `sflow`
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+<Callout variant="important">
```

Comment:
Should it be "For JFlow Juniper Networks"?

Reply:
I'll add a possessive apostrophe 👌🏻